app/fixtures/ember/release.js

```diff
@@ -13,5 +13,5 @@ export default {
   enableTestURL: true,
   debugFileName: ".debug.js",
   ignoreFiles: ['ember.js'],
-  installWithEmberCLI: '# Install Ember %s:\nbower install --save ember#v%s\n# Or, install the latest build of this channel which may include unreleased incremental changes:\nbower install --save ember#release'
+  installWithEmberCLI: '# Install Ember %s:\nnpm install --save-dev ember-source@v%s\n# Or, install the latest build of this channel which may include unreleased incremental changes:\nnpm install --save-dev emberjs/ember.js#release'
```

Comment:
Lets remove the channel specific instructions here (they won't work due to lacking prebuild assets).